src/pq_flash_index.cpp

```diff
@@ -41,7 +41,7 @@ namespace diskann
 
 template <typename T, typename LabelT>
 PQFlashIndex<T, LabelT>::PQFlashIndex(std::shared_ptr<AlignedFileReader> &fileReader, diskann::Metric m)
-    : reader(fileReader), metric(m)
+    : reader(fileReader), metric(m), memory_in_bytes(0)
```

Comment:
Are you considering scratch space allocated inside setup_thread_data()?

Reply:
yes. I also could add that. but even though I add that, @harsha-simhadri  may still have concern(get accurate value and count the new memory for the new code). So I may wait for the final conclusion to continue the change.